### ISSUE TITLE
any way I can help out?

### ISSUE THREAD

**Issue by mainrs** [Wilyb/Wily-bit-s-GTNH-tex#2]
Hey! 

Do you have a vision for the higher tiered electronic components? A color palette for example, since they all use the same texture!

**Comment by liviuslivius** [comment]
Hey!

I really enjoy playing with the pack since u guys helped me setting it up haha. I love all the textures. How does the vision for the future look. Can i help in any way? Just trying to support a pack i love

**Comment by Wilyb** [comment]
oh hi, i am not someone who is used to using github at all haha, i am mostly just doing this texturepack as i play the modpack so i kinda just make the designs as i go, usually waiting for materials to bulk process
what do you mean by higher tiered electronic components? you mean the usual stuff like sensors and motors? the resistors and capacitors? the circuits? or what?

**Comment by Wilyb** [comment]
oops sorry, github is rather confusing to me still

**Comment by mainrs** [comment]
> what do you mean by higher tiered electronic components? you mean the usual stuff like sensors and motors? the resistors and capacitors? the circuits? or what?

Yep! I am currently not that far into the game (on my way to MV!) and I didn't notice that higher-tiered components like motors, pistons, etc. have different colors already.

So I assume you kind of stick to the existing color scheme of these crafting items?

**Comment by Wilyb** [comment]
i actually base their colors on their material components, its why aluminum in my texture pack is blue, but in vanilla gt its orange

**Comment by mainrs** [comment]
That makes sense, thank you!
